nservicebus/logging/index_level_core_[6.0,).partial.md

```diff
@@ -0,0 +1,29 @@
+The supported logging levels are
+
+ * Debug
+ * Info
+ * Warn
+ * Error
+ * Fatal
+
+
+### Changing the Logging Level
+
+Note: When logging level is defined in both app.config and code the code wins.
+
+Warning: Configuration of logging via `app.config` is not recommended in Version 6, use the [code configuration API](#logging-levels-changing-the-logging-level-via-config-api) instead. The configuration section will be removed in Version 7.
```

Comment:
Logging and encryption are the only ones that had a partial that needed changing. The rest show for all the versions. This and encryption only appear if the user selects V6 from the version dropdown.